examples/pysdl2.py

```diff
@@ -0,0 +1,357 @@
+"""
+ Simple SDL2 / cefpython3 example.
+
+ Only handles mouse events but could be extended to handle others.
+
+ Requires pysdl2 (and SDL2 library).
+
+ Install instructions.
+ 
+ 1. Install SDL libraries for your OS, e.g:
+ 
+    Fedora:
+    
+        sudo dnf install SDL2 SDL2_ttf SDL2_image SDL2_gfx SDL2_mixer
+        
+    Ubuntu:
+    
+        sudo apt-get install libsdl2-dev
+
+ 2. Install PySDL via PIP:
+
+    sudo pip2 install PySDL2
+
+ Tested configurations:
```

Comment:
Let's put Tested configurations just below "Simple SDL2 / cefpython3 example." text. There are many instructions and notes and list of tested configurations is more important to display first.